dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/AuditOperationType.java

```diff
@@ -17,46 +17,86 @@
 
 package org.apache.dolphinscheduler.common.enums;
 
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
+
+import lombok.Getter;
 
 /**
  * Audit Operation type
  */
+@Getter
 public enum AuditOperationType {
 
-    CREATE(0, "CREATE"),
-    READ(1, "READ"),
-    UPDATE(2, "UPDATE"),
-    DELETE(3, "DELETE");
+    CREATE(0, "Create", false, false),
```

Comment:
It's better to combine the AuditOperationType with AuditObjectType into one enum. e.g. PROJECT_CREATE, PROJECT_DELETE, PROJECT_UPLOAD...